src/GafferSceneUI/TranslateTool.cpp

```diff
@@ -116,6 +116,12 @@ void TranslateTool::updateHandles()
 {
 	handles()->setTransform( handlesTransform() );
 
+	// Because we provide multiple orientations, the handles
+	// may well not be aligned with the axes of the transform
+	// space. So any given handle might affect several components
+	// of the target translation. For each handle, check to see
+	// if each of the plugs it effects are settable, and if not,
+	// disable the handle.
```

Comment:
I'm willing to accept this for now, but I don't expect a user will. The simplest case (e.g. translate.x is driven by something, but I don't care, I just want to move translate.y) means I can't move anything. It kind of sounds like the enabled state of the handle should be taking the orientation plug into account?